src/components/App/App.tsx

```diff
@@ -162,10 +162,17 @@ const App = () => {
               return null;
             }}
           />
-
           <Route path="/gift-a-meal-home">
             {returnComponent(Page.GiftAMeal)}
           </Route>
+          <Route
+            path="/cookbook"
+            component={() => {
+              window.location.href =
+                'https://www.sendchinatownlove.com/cookbook';
+              return null;
+            }}
+          />
```

Comment:
Do we need this?
Don't we add the external link in the NavBar.tsx like
``` {
      url: 'https://www.friendofafriend.studio/shop',
      translation: 'friendofafriend-studioxscl',
      external: true,
    },

Reply:
yep; pushed